client/log_verifier.go

```diff
@@ -40,6 +41,27 @@ func NewLogVerifier(hasher hashers.LogHasher, pubKey crypto.PublicKey) LogVerifi
 	}
 }
 
+// NewLogVerifierFromTree creates a new LogVerifier
```

Comment:
... using the algorithm set specified by config.  ?

Reply:
yep. Added comment.